packages/apps/explorer/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@kadena/block-explorer",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prisma:generate": "npx prisma generate",
+    "create:info:test": "curl --request GET 'http://web:3000/api/info/create?network=Testnet'",
+    "create:info:main": "curl --request GET 'http://web:3000/api/info/create?network=Mainnet'"
+  },
```

Comment:
- A `"test"` command needs to be added as it's needed by `rush`.
- Can we add `lint-staged` with `lint-staged` config. (see other packages for examples)